corehq/apps/geospatial/static/geospatial/js/case_grouping_map.js

```diff
@@ -17,11 +17,65 @@ hqDefine("geospatial/js/case_grouping_map",[
         self.coordinates = coordiantes;
         self.caseLink = caseLink;
 
+        // TODO: Group ID needs to be set
+        self.groupId = null;
+
         return self;
     }
 
+    function exportModel() {
+        var self = {};
+
+        self.casesToExport = ko.observableArray([]);
+
+        self.handleExportCSV = function () {
```

Comment:
nit: I believe a better name could be `downloadCSV`.

Reply:
See [this](https://github.com/dimagi/commcare-hq/pull/33610#discussion_r1358076447) comment for changes.